### PR TITLE
quick style refactor

### PR DESCRIPTION
**Pull request by michaelshobbs**
- refactor function defs to be posix-like
 - remove usage of force positional argument in tests
 - support --force like arguments in client
 - implements parallel test execution in CircleCI

### CODE REVIEW

File: contrib/dokku_client.sh [hunk 1]
```diff
@@ -98,9 +98,24 @@ if [[ ! -z $DOKKU_HOST ]]; then
...
+          if [[ "$arg" =~ --* ]];then
+            long_args+=" $arg"
+            args=(${args[@]/$arg})
+          else
+             verb="$arg"
+          fi
+        done
+      fi
+    fi
+    ssh "dokku@$DOKKU_HOST" $long_args "$verb" "$appname" "$args"
```
Comment (michaelshobbs) [inline contrib/dokku_client.sh]: definitely open to a better way of doing this. its the client, so I kind of just hacked it for now

**Comment by josegonzalez** [comment]
Looks good other than failed linting :)

**Comment by michaelshobbs** [comment]
@josegonzalez you good with this?

**Comment by michaelshobbs** [comment]
I'm going to assume 3x increase in speed is a good thing. :imp:

**Pull request merged**